src/main/scala/sbtorgpolicies/settings/bash.scala

```diff
@@ -94,6 +96,19 @@ trait bash extends bashCompat {
     }
   }
 
+  private[this] def isLastScalaBinaryVersion(scalaBinaryV: String, crossV: Seq[String]): Boolean = {
+
+    val MajorMinorPatchR = "(\\d+\\.\\d+)\\.\\d+".r
+
+    def majorMinorV(v: String): Option[String] = v match {
+      case MajorMinorPatchR(majorMinorV) => Some(majorMinorV)
+      case _                             => None
+    }
+
+    crossV.lastOption.flatMap(majorMinorV).exists(_ == scalaBinaryV)
```

Comment:
Is there a chance that someone/we would build artifact(s) for multiple patch version?
2.12.1/2.12.2/2.12.3 ... Would `sbt release` run for each version, if labelled in `travis.yml`?

Does `crossV`/`crossScalaVersions` require any order of Scala versions? (`lastOption`)

Reply:
Yes, it will run for each version, but we could put it in the docs. IMHO it's a minor drawback compared to the current issue but we can discuss it here.

Good point, I think I could add a `sorted` call before the `lastOption` to be sure we get the major version.

Reply:
Makes sense, having multiple patch versions seems like a very special case.

`sorted` sounds good 👍

Reply:
Now I'm thinking loudly, but, another approach could be to parse the Travis file. In that way we could know not only the major Scala version but also the major SBT version.